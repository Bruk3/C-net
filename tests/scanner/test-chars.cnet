'\r'
'\f'
'\t'
'\n'
'\0'
'\''