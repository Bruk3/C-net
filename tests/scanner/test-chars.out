CHARLIT(13)
CHARLIT(12)
CHARLIT(9)
CHARLIT(10)
CHARLIT(0)
CHARLIT(47)
EOF
